<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisionWidgetClass</class>
 <widget class="QMainWindow" name="VisionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisionWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="VisionView" name="visionView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>640</width>
      <height>640</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImageBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagePathEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="VisionViewToolBox" name="visionViewToolBox" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1346</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VisionView</class>
   <extends>QLabel</extends>
   <header>visionview.h</header>
  </customwidget>
  <customwidget>
   <class>VisionViewToolBox</class>
   <extends>QWidget</extends>
   <header>visionviewtoolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="visionwidget.qrc"/>
 </resources>
 <connections/>
</ui>
